lib/scalaplugin.py

```diff
@@ -41,51 +36,37 @@ def scala_version_dep(version):
     return "org.scala-lang:scala-compiler:{}".format(version)
 
 
-def get_bloop_artifacts():
-    version = "2.12.8"
-    # The version of bsp4s that bloop depeneds on isn't published so we override it
-    deps = ["ch.epfl.scala::bloop-frontend:1.2.5", "ch.epfl.scala::bsp4s:2.0.0-M3"]
-    expanded = [expand_scala_dep(version, dep) for dep in deps]
-    allDeps = expanded + [scala_version_dep(version)]
-    return allDeps
-
-
-def bloop_classpath(coursier, cache, offline=True):
-    deps = get_bloop_artifacts()
-    offlineArgs = ["-m", "offline"] if offline else []
-    cmd = [coursier, "fetch"] + offlineArgs + ["--classpath", "--cache", cache] + deps
-    proc = subprocess.run(cmd, stdout=subprocess.PIPE, universal_newlines=True)
-    return proc.stdout.rstrip()
-
-
-def run_bloop(coursier, bloop_home, cache, args):
-    classpath = bloop_classpath(coursier, cache)
-    if classpath is None:
-        return 1
-    arglist = args.split()
-    set_home = "-Duser.home={}".format(bloop_home)
-    cmd = ["java", "-Xss8M", set_home, "-cp", classpath, "bloop.Cli"] + arglist
-    proc = subprocess.run(cmd)
-    return proc.returncode == 0
-
-
-def fetch_scala_compiler_bridge(coursier, bloop_home, cache, version):
-    # We need absolute paths here
-    cache = os.path.abspath(cache)
-    bloop_home = os.path.abspath(bloop_home)
-    classpath = bloop_classpath(coursier, cache, offline=False)
-    if classpath is None:
-        return 1
-    fetcher = "{}/scala-bridge-fetcher_2.12-0.1.0.jar".format(SCRIPT_DIR)
-    classpath = classpath + ":" + fetcher
-    # Make sure bloop_home exists
-    os.makedirs(bloop_home, mode=0o755, exist_ok=True)
-    set_home = "-Duser.home={}".format(bloop_home)
-    cmd = ["java", set_home, "-cp", classpath, "sifive.ScalaCompilerBridgeFetcher", version]
-    # This creates a target directory, put it in bloop_home
-    proc = subprocess.Popen(cmd, cwd=bloop_home)
-    proc.wait()
-    return proc.returncode == 0
+def make_jar(jarfile: str, dirs: List[str]) -> None:
```

Comment:
This looks new, but it comes from `bloop_wake`. Now that you shouldn't review it, but it's been in use the whole time.